### PR TITLE
Make db_postgres.open() consistent

### PR DESCRIPTION
**Pull request by lyndsysimon**

### CODE REVIEW

File: lib/impure/db_postgres.nim [hunk 1]
```diff
@@ -519,7 +519,13 @@ proc open*(connection, user, password, database: string): DbConn {.
   ##
   ## Note that the connection parameter is not used but exists to maintain
```
Comment (Araq) [inline lib/impure/db_postgres.nim]: Update the documentation too please.

Comment (lyndsysimon) [reply lib/impure/db_postgres.nim]: I've removed the note about the unused param.

**Comment by lyndsysimon** [comment]
I'd be happy to @dom96 - in fact, the reason I didn't add a basic example already is that none of the other DB libraries have basic examples within the `open` method. If I add it here, I'd want to add it to the others as well.

Would it be more helpful if I instead took some time and aligned the documentation for all of them? I noticed that this file doesn't have a "Larger Example" section, for instance.

**Comment by dom96** [comment]
Yes, please do so. But don't create a "larger example". All examples should be independent, concise and to the point. Sort of like the docs in the [``httpclient`` module](https://nim-lang.org/docs/httpclient.html)

**Pull request merged**